tests/codegen/i128-x86-callconv.rs

```diff
@@ -0,0 +1,79 @@
+//! Verify that Rust implements the expected calling convention for `i128`/`u128`.
+
+// Eliminate intermediate instructions during `nop` tests
+//@ compile-flags: -Copt-level=1
+
+//@ add-core-stubs
+//@ revisions: MSVC MINGW
+//@ [MSVC] needs-llvm-components: x86
+//@ [MINGW] needs-llvm-components: x86
+//@ [MSVC] compile-flags: --target x86_64-pc-windows-msvc
+//@ [MINGW] compile-flags: --target x86_64-pc-windows-gnu
+//@ [MSVC] filecheck-flags: --check-prefix=WIN
+//@ [MINGW] filecheck-flags: --check-prefix=WIN
```

Comment:
What is the point of this `--check-prefix=WIN`? The test can just use the `CHECK` prefix for annotations that should be checked in all revisions.

Reply:
I just did this so we could could add non-windows targets in the same file in the future.

Reply:
...which I see you did in https://github.com/rust-lang/rust/pull/137094 :)

Reply:
Ah, great, I did exactly that in https://github.com/rust-lang/rust/pull/137094. :)  However I had to rediscover this as it wasn't clear at all from the test.  `filecheck-flags: --check-prefix` is a super neat trick for making these tests nicer, but I have no idea how anyone is supposed to know about that... it doesn't seem mentioned in the dev guide either?

Reply:
Tbh I only thought to try it because I noticed LLVM's tests always set `--check-prefix`, seems like it's barely used in our suite since bootstrap does it for revisions automatically. I think this hasn't been done much in the past because `#![no_core]` is pretty awkward to use, but since minicore was added recently this is much better.